errors.js

```diff
@@ -0,0 +1,19 @@
+exports.handlePSQErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "bad request" });
+  } else {
+    next(err);
+  }
+};
+
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
```

Comment:
As we're not currently using these first 2 error handlers, they shouldn't be here for now. Nice thinking ahead though, as they'll definitely be returning later! 👍